native~/Editor/src/CesiumIonSessionImpl.cpp

```diff
@@ -657,6 +711,13 @@ const std::vector<CesiumIonClient::Token>& CesiumIonSessionImpl::getTokens() {
   }
 }
 
+const CesiumIonClient::ApplicationData& CesiumIonSessionImpl::getAppData() {
+  if (this->_appData) {
+    return *this->_appData;
+  }
+  return CesiumIonClient::ApplicationData();
```

Comment:
You can conveniently simplify this with `value_or`!

```suggestion
  return this->_appData.value_or(CesiumIonClient::ApplicationData());
```

Reply:
Also it looks like this is initialized in the function body here, but is `static const` in Cesium for Unreal. Is there something preventing the two implementations from being the same?